terraform/modules/ecr/main.tf

```diff
@@ -0,0 +1,23 @@
+terraform {
+  required_version = "~>1.5.1"
+
+  required_providers {
+    aws = {
+      source  = "hashicorp/aws"
+      version = "~>5.0"
+    }
+  }
+}
+
+#tfsec:ignore:aws-ecr-enforce-immutable-repository:we require mutable tags for our current deployment process
+#tfsec:ignore:aws-ecr-repository-customer-key:encryption using KMS CMK not required
+resource "aws_ecr_repository" "this" {
+  #checkov:skip=CKV_AWS_51:we require mutable tags for our current deployment process
+  #checkov:skip=CKV_AWS_136:encryption using KMS not required

```

Comment:
I assume there's no way around needing to repeat the skip / ignore comments in slightly different places? It would be nicer if we could group ones covering the same thing, but if they need to be in different spots then so be it. (It is a major annoyance about using both though)

Reply:
I believe the only way around avoiding this sort of repetition is by maybe providing the ignore/skip statements at a higher level where appropriate (e.g. where the module is being called from)

I think in general we will aim to put all ignore/skip statements on the module declaration itself, so that anything that calls it won't need to provide such statements

I think this is more of an issue with the way the code is structured for creating the backend, but the above approach I think will serve fine going forward for the rest of the infra

We could create a wrapper for the 3 S3 buckets declared in backend/main.tf, and put the statements on the wrapper, but I don't think it's worth all the work. Are you happy for us to leave the backend as is?